### ISSUE TITLE
Using Split in a project where it has worked in the past, it does not do anything

### ISSUE THREAD

**Issue by muradjames** [microsoft/azure-boards-split#34]
Using Split in a project where it has worked in the past, it does not create a new work item or split any child work items out - it essentially does nothing. Is there a way to put it in "debug" mode or get some feedback on what is happening or generate some logs of what the problem might be? It has saved us so much time and pain in the past :)

**Comment by AminTi** [comment]
I was unable to reproduce the issues. Could you please check if there are any relevant errors or warnings in the console log or network log? This information could help identify the problem

**Comment by muradjames** [comment]
Hi Amin - Thanks for your reply - why did I not think to check the console? 🤦‍♂️
I found the problem - I had added a new boolean field to the work item in the process template. 
It defaulted to false but only a few work items had a value in it at all - for most of them it was blank. 
The split extension was complaining about a rule failing for that field because booleans are mandatory but most of them had no value set so that was stopping it working. 
I did a bulk update to set the blanks to false and it's fine now.
Thank you.
